src/services/print.js

```diff
@@ -550,6 +549,24 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the capabilities of the report
+ * @param {function()} success Success callback
+ * @param {function()=} opt_error Error callback
```

Comment:
These callbacks are not needed. The `getCapabilities` returns the http promise so the user can just call `success`, `error `or `then` on the promise.